docs/pages/versions/unversioned/sdk/firebase-recaptcha.md

```diff
@@ -83,29 +85,32 @@ import { getAuth, PhoneAuthProvider } from 'firebase/auth';
   // ignore app already initialized error in snack
 }*/
 
+// Firebase references
+const app = getApp();
+const auth = getAuth();
+
+// Double-check that we can run the example
+if (!app?.options || Platform.OS === 'web') {
+  throw new Error('This example only works on Android or iOS, and requires a valid Firebase config.');
+}
+
 export default function App() {
+  // Ref or state management hooks
   const recaptchaVerifier = React.useRef(null);
   const [phoneNumber, setPhoneNumber] = React.useState();
   const [verificationId, setVerificationId] = React.useState();
   const [verificationCode, setVerificationCode] = React.useState();
-  const app = getApp();
+
   const firebaseConfig = app ? app.options : undefined;
-  const [message, showMessage] = React.useState(
-    !firebaseConfig || Platform.OS === 'web'
-      ? {
-          text:
-            'To get started, provide a valid firebase config in App.js and open this snack on an iOS or Android device.',
-        }
-      : undefined
-  );
+  const [message, showMessage] = React.useState();
   const attemptInvisibleVerification = false;
 
   return (
     <View style={{ padding: 20, marginTop: 50 }}>
       <FirebaseRecaptchaVerifierModal
         ref={recaptchaVerifier}
-        firebaseConfig={firebaseConfig}
-        attemptInvisibleVerification={attemptInvisibleVerification}
+        firebaseConfig={app.options}
+        // attemptInvisibleVerification
```

Comment:
Is it necessary to leave the commented out prop in here?

Reply:
In this example, I'm mostly trying to steer people to try out both. Adding a comment was too long, but just leaving this there should be enough for people to try both (I think)